e2e/condition_schedule_test.go

```diff
@@ -128,11 +176,42 @@ func TestCondition_Schedule_Basic(t *testing.T) {
 			api.WaitForEvent(t, cts, taskName, registerTime, scheduledWait)
 			checkScheduledRun(t, taskName, registerTime, taskSchedule, port)
 
-			// confirm resources created
+			// confirm service resources created
 			resourcesPath := filepath.Join(tempDir, taskName, resourcesDir)
 			testutils.CheckFile(t, true, resourcesPath, "api-1.txt")
 			testutils.CheckFile(t, true, resourcesPath, "web-1.txt")
 
+			// Check KV events
+			if tc.isConsulKV {
```

Comment:
I don't "love" having this branching path here, but I figured that this can be resolved when someone takes on the e2e refactoring we've talked about.